### PR TITLE
[issue#7] ADD rhyu-class.js file

### LINKED ISSUE #7: Class thats greets user
**Is your feature request related to a problem? Please describe.**
the user is not being greeted 

**Describe the solution you'd like**
to add a class to greet them

**Describe alternatives you've considered**
making a function 

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by rhyusarmiento**
created rhyu-class.js to src folder fixed #7

### CODE REVIEW

File: app/rhyu-class.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+	constructor(userName = 'default') {
+		this.name = userName;
+	}
+	greeting() {
+		return `${this.name} hello`
+	}
+	emergencyWarning() {
+		return `GET OUT ${this.name}`
+	}
+}
```
Comment (gleekzorp) [inline app/rhyu-class.js]: Please delete this file.

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Please remove the `app/rhyu-class.js` and also test your `rhyu-cass.js` inside the `bootstrap.js` file.

File: app/rhyu-class.js (hunk 1 above)
Comment (rhyusarmiento) [reply app/rhyu-class.js]: working on this thanks

**Review by gleekzorp — APPROVED** [review]
Awesome!

**Pull request merged**